rose/web/game.js

```diff
@@ -17,16 +17,21 @@ var ROSE = (function() {
         this.controller = new Controller();
         this.rate = new Rate([0.5, 1.0, 2.0, 5.0, 10.0]);
 
-        var loader = new ImageLoader(function() {
+        var imageLoader = new ImageLoader(function() {
```

Comment:
I would like to avoid mixedCase names in our code, they are more error prone and harder to read and write. Can rename this to image_loader?